src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+      />
+      <h2 class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <span class="card__product-code">Product code: 195434</span>
+      <div class="card__rating-box">
+        <ul class="stars stars--3">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <span class="card__reviews">Reviews: 5</span>
+      </div>
+      <div class="card__price-box">
+        <span class="card__price-label">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </div>
+      <button
```

Comment:
Format your code for better readability
<img width="1061" alt="Знімок екрана 2024-08-13 о 17 32 59" src="https://github.com/user-attachments/assets/aa784e9b-0dc4-45f3-8567-395de3e7e992">

```suggestion
      />
      
      <h2 class="card__text">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <span class="card__product-code">Product code: 195434</span>
      
      <div class="card__rating-box">
        <ul class="stars stars--3">
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
        </ul>
        
        <span class="card__reviews">Reviews: 5</span>
      </div>
      
      <div class="card__price-box">
        <span class="card__price-label">Price:</span>
        <span class="card__price-value">$2,199</span>
      </div>
      
      <button
```

Reply:
I cant do that my tests gives me error because of that as I have such a plagin which I cant delete.